tests/transforms/test_optimization/test_push_hadamards.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2018-2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the `zx.push_hadamards` transform.
+"""
+import numpy as np
+import pytest
+
+import pennylane as qml
+from pennylane.tape import QuantumScript
+
+
```

Comment:
Can we also `importorskip` here? This might be helpful for devs who used to test locally running the whole tests base